network/wsPeer.go

```diff
@@ -486,6 +486,20 @@ func (wp *wsPeer) readLoop() {
 			// network maintenance message handled immediately instead of handing off to general handlers
 			wp.handleFilterMessage(msg)
 			continue
+		// the remaining valid tags: no special handling here
+		case protocol.AgreementVoteTag:
+		case protocol.NetPrioResponseTag:
+		case protocol.PingTag:
+		case protocol.PingReplyTag:
+		case protocol.ProposalPayloadTag:
+		case protocol.StateProofSigTag:
+		case protocol.TxnTag:
+		case protocol.UniCatchupReqTag:
+		case protocol.UniEnsBlockReqTag:
+		case protocol.VoteBundleTag:
```

Comment:
we need that exhaustive switch statement linter

Reply:
that would be cool, but these tags are not defined in an enum... idk if there is something to lint switching on const Tag variable assignments